src/app/store.ts

```diff
@@ -1,13 +1,19 @@
 import { configureStore, ThunkAction, Action } from '@reduxjs/toolkit';
 // eslint-disable-next-line import/no-cycle
 import counterReducer from '../features/counter/counterSlice';
+import usersReducer, { fetchUsers } from '../features/users/usersSlice';
+import postsReducer from '../features/posts/postSlice';
 
 export const store = configureStore({
   reducer: {
     counter: counterReducer,
+    users: usersReducer,
+    posts: postsReducer,
   },
 });
 
+store.dispatch(fetchUsers());
```

Comment:
The `fetchUsers` action is dispatched immediately after the store is created. Ensure this is the intended behavior and aligns with the application's logic, as it will trigger a fetch operation as soon as the application starts.